
\begin{savequote}[85mm]
``As you think, so you become. Our busy minds are forever jumping to conclusions, manufacturing and interpreting signs that arenâ€™t there.``
\qauthor{--- Epictetus, \textit{The Enchiridion}}
\end{savequote}


\chapter{Conclusion}
\label{ch:conclusion}

This thesis explored different computational methods to assess the genetic heterogeneity of multiple patient samples DNA sequencing. With sequencing costs now trending towards \$100 per genome, many clinical studies are now accumulating data at an unprecedented rate \cite{Stephens2015}, but computational methods have not kept up with the development. With multiple spatial and longitudinal samples sequenced, the known concept of spatial and tissue heterogeneity could be assessed and insights into disease trajectory and evolution generated. In the past, the accurate molecular diagnosis of a patients cancer led to the discovery of targeted therapies and a massive improvement in cancer care. So it is likely, that similar advances can be made with the accurate analysis of the diversity and history of cancer clones within a patient. While single cell sequencing has already highlighted and described new cell states and types, the methods are far from being able to be used in a clinical situation. Additionally, the amount of data already generated for collaborative efforts like TCGA \cite{IPCAWGC2020} or PCAWG are cause enough to optimise and develop methods to facilitate further research in the cancer space.

The contributions of this work include the development of multiple proof of principle methods, which show that the analysis of bulk sequencing requires further research and has unrealised potential for both diagnostic and research questions.

This thesis presents three distinct but related projects, which explore the analysis of tumour heterogeneity at different levels and depth, with a focus on method development.

In \autoref{ch:variantcalling} we presented the work we conducted to improve the detection of somatic variants at very low allele frequencies. When multiple samples, separated in time or space, of the same patient were available, we were able to improve the detection threshold substantially. These low abundance variants are invaluable in a clinical setting, where they can indicate an arising resistance mechanism, or relapse of disease. With the improved sensitivity of our method, treatment of patients can be adjusted earlier and more accurately. With the increase in multi-region analysis in cancer research, several research projects are already using the joint somatic variant calling approach developed. In the future these concepts could be adjusted for the use in single cell sequencing and to employ evolutionary modelling to allow usage of priors for the spatial or temporal distance of samples which are analysed jointly.

\autoref{ch:cascade} illustrates the in-depth analysis of resistance mechanisms and evolutionary history of five lung cancer patients enrolled in the CASCADE autopsy program. Using the joint somatic variant calling from \autoref{ch:variantcalling} as a basis, we explored various ways to describe and visualise the disease trajectory of each patient from diagnosis till death. Additionally to the variants, we used copy number analysis and structural variants to contrast and compare each sample within a patient to generate phylogenies to visualise the evolutionary distances and to generate a pseudo time scale for the timing of mutations. In order to further clarify the grouping and distances of samples, we implemented a distance measure based on mitochondrial variants. With this method we could assess the effect of selection pressure on established variants and their timing in the nuclear variant derived phylogenies. This work has already lead to two publications, describing two resistance mechanisms: a novel resistance mechanism to a RET-fusion targeted treatment in patient CA-A \cite{Solomon2020}, and the mechanism of small cell transformation in patient CA-L \cite{Burr2019}. Across the cohort with unsurprisingly found few unifying features apart from loss of heterozygosity and large scale genomic amplification. Clear genomic determinants of treatment resistance were identified for the three NSCLC cases which did not show phenotypic switching. The diversity of these genomic mechanisms were profoundly highlighting the true extend of inter patient heterogeneity. In contrast to the NSCLC cases, the two cases with small cell transformation showed distinct evolutionary trajectories , with similarity in their phylogenies, both nuclear and mitochondrial, suggesting initial shared evolution with high private mutations and no clear genetic discriminant for the small cell transformation. Additionally, the small cell transformations did not exhibit the previously hypothesised genomic hallmarks of TP53 and RB1 loss. These findings moved to focus on the importance of characterising non-genomic evolution in parallel to genomic changes in the study of treatment resistance. To fully explore the heterogeneity of disease in these patients and generate a complete landscape, further RNA and epigenetic profiling will be necessary to shed light on the mechanism leading to small cell phenotypic switching.

With \autoref{ch:mmf} we explored the avenue of measuring tumour evolution over time through the use of ctDNA, as it is often not feasible to  continuously biopsy a patient during treatment. We tailored a method to be fully tumour agnostic which can be easily be applied to the clinical setting by using low coverage whole genome sequencing, which has already been used for diagnostics in some cancers. The method uses highly specialised filtering steps to eliminate the background noise from the ``normal contamination`` and sequencing errors in these data. We showed that the method can accurately detect specific cancer related signatures at low tumour purity and tumour burden in simulated and patient data for melanoma and breast cancer. MisMatchFinder was able to detect signatures, which were not causing clonal fixation, but rather were present in all clones at different levels. This same effect was also observed when we built a machine learning model to predict the presence or absence of cancer in a plasma sample. Tumour samples were accurately classified without their commonly associated mutational signatures and instead showed substantially perturbed mutational patterns. The predictive ability of MisMatchFinder was already comparable to the current gold standard ichorCNA, and there are multiple possible ways to improve MisMatchFinder further. The restriction of the analysis to fragments of a size which enriches for tumour signal showed substantial performance improvements in our data and recent literature suggests a similar effect for fragment start sites and sequence. These additional filters should increase sensitivity and help reduce the amount of false positive classifications in our predictions. Additionally, MisMatchFinder and predictor could be extended to use double base substitutions and InDel signatures, which are known to be very specific for certain cancer types. Lastly, the classifier could be retrained to classify cancer type in addition to cancer presence with additional data from public datasets.

With the introduction of further sequencing platforms and technologies apart from Illumina \cite{SingularGenomics2021,UltimaGenomics2022,ElementBiosciences2022}, the ability to generate high quality multi-region/-sample sequencing datasets is more accessible than ever before. These new datasets will require ongoing development of computational tools to analyse the amount of data, learning from the methods already deployed in other fields like population genetics. Ever growing cancer cohort studies will require substantial optimisation and development of methods as can be seen with the rapid development of single cell analysis. 

These methods will also require investment of time and money into generation of gold standard datasets to further the improvement of algorithms with objective performance measures. As discussed in this thesis, while simulated datasets are ideal to test certain aspects of methods, approaches like ``Genome in a Bottle`` are necessary for somatic variant calling methods to continue improving inn the same way that germline variant calling field has evolved \cite{ValleInclan2022}.

While somatic variant calling in cancer often has the flair of being ``solved`` every improvement in the field leads to further understanding of concepts and biological processes. In particular, the field of multi-sample variant calling analysis is still very young and requires new guidelines and best practices to evolve, as it challenges current knowledge and hypothesis. The contribution of this work to the field was focused on methods to maximise insight into cancer biology in a multitude of cases. We showed that jointly calling somatic variants is superior and able to uncover previously unappreciated genetic variation. Secondly, we contributed to the biomedical field by detecting and describing novel modes of resistance and lastly, we developed a proof of concept approach to detect somatic mutational patterns from plasma samples. Overall, we believe the work done during this PhD has significantly contributed to the knowledge and capabilities in the field of genetic cancer heterogeneity.

