
\begin{savequote}[85mm]
``As you think, so you become. Our busy minds are forever jumping to conclusions, manufacturing and interpreting signs that arenâ€™t there.``
\qauthor{--- Epictetus, \textit{The Enchiridion}}
\end{savequote}


\chapter{Conclusion}
\label{ch:conclusion}

This thesis explored different computational methods to assess the genetic heterogeneity of multiple patient samples DNA sequencing. With sequencing prices trending towards \$100 per genome, many clinical studies accumulate data at an unprecedented rate \cite{Stephens2015}, but computational methods have not kept up with the development. With spatial or longitudinal multi samples sequencing, the known concept of spatial and tissue heterogeneity could be assessed and insights into disease trajectory and evolution generated. In the past, the the accurate molecular diagnosis of a patients cancer led to the discovery of targeted therapies and a massive improvement in cancer care. So it is likely, that similar advances can be made with the accurate analysis of the diversity and history of cancer clones within a patient. While single cell sequencing has already highlighted and described new cell states and types, the methods are far from being able to be used in a clinical situation. Additionally, the amount of data already generated for collaborative efforts like TCGA \cite{IPCAWGC2020} or PCAWG are cause enough to optimise and develop methods to further research in the cancer space.

The contributions of this work include the development of multiple proof of principle methods, which show that the analysis of bulk sequencing requires further research and has unrealised potential for both diagnostic and research questions.

This thesis presents three distinct but related projects, which explore the analysis of tumour heterogeneity at different levels and depth, with a focus on method development.

In \autoref{ch:variantcalling} we presented the work we conducted to improve the detection of somatic variants at very low allele frequencies. When multiple samples, separated in time or space, of the same patient are available, we were able to improve the detection threshold substantially. These low abundance variants are invaluable in a clinical setting, where they can indicate an arising resistance mechanism, or relapse of disease. With the improved sensitivity of our method, treatment of patients can be adjusted earlier and more accurately. With the increase in multi-region analysis both in the field and in this institute, several research projects already use the joint somatic variant calling approach developed, but in the future these concepts should be adjusted for the use in single cell sequencing and an evolutionary model to allow usage of priors for the spatial or temporal distance of samples which are analysed jointly.

\autoref{ch:cascade} illustrates the in-depth analysis of resistance mechanisms and evolutionary history of five lung cancer patients enrolled in the CASCADE autopsy program. Using the joint somatic variant calling from \autoref{ch:variantcalling} as a basis, we explored various ways to describe and visualise the disease trajectory of each patient from diagnosis till death. Additionally to the variants, we used copy number analysis and structural variants to contrast and compare each samples within a patient to generate phylogenies to visualise the evolutionary distances and to generate a pseudo time scale for the timing of mutations. In order to further clarify the grouping and distances of samples, we implemented a distance measure based on mitochondrial variants. With this method we could assess the effect of selection pressure on established variants and their timing in the nuclear variant derived phylogenies.

\todo[color=red,inline]{fill this}

heterogeneity is a known concept, but methods to measure and quantify are lacking

Joint variant calling is mandatory when using multiple samples (which is the only way to do heterogeneity)
Even in a world of single cell, old data and clinical data will not likely be single cell soon.

Mitochondrial variants allow assessment of temporal history due to their quicker mutation rate, small cell not solved yet

MisMatchFinder allows measurement of tumour heterogeneity over time/ evolution with no prior knowledge in a clinical background (very cheap)

Good step forward, but many missing heterogeneities (epigenetic and transcriptional) longitudinal only scraped with methods being applicable -> downstream analysis is still lacking with the amount of data available 