\section{Introduction}
\label{variantcalling-sec:intro}
In 2018, at the start of the work presented in this thesis, we observed a difference in methodology  between germline and somatic variant calling methods. Where all ``modern``\note{unified quotation marks} germline variant callers, like Strelka2 \cite{Kim2018}, HaplotypeCaller \cite{Poplin2017}, DRAGEN \cite{Miller2015} and DeepVariant \cite{Poplin2018},  have the built-in capability to jointly call multiple related samples, for example, from family trios, virtually no somatic variant caller had this functionality. 

The joint analysis of smaller cohorts improves the performance of germline variant calling methods significantly by allowing to assess technical artefacts, which might be unique for the individual sequencing machine or the researcher handling the DNA \cite{Schirmer2016,Stoler2021}. Additionally, as certain parts of the genome are more problematic to sequence (\autoref{intro-sec:sequencing}) and map (\autoref{intro-sec:mapping}), a ``control`` sample can help to distinguish if a certain observed change \add{frequently} occurring \remove{frequently} is a technical issue or, in fact, a real change.

For somatic variant calling, this concept has been adopted \remove{on} in the genome analysis toolkit (GATK) \cite{BrianOConnor2020} to allow the use of a panel of normals (PON), which contain frequently seen changes in healthy (``normal``) individuals analysed with the same sequencing technology \cite{GATKTeam2021}. Although, in contrast to the  more intricate model for the germline equivalent, this is a post-processing step of the analysis. Mutect2, \remove{which is} the most recent somatic variant calling algorithm provided by the Broad institute, also provides a multi-sample mode, for which all tumour samples need to be from the same patient, either related longitudinally or spatially \cite{GATKTeam2020}. However, this mode is not very well publicised, and all tutorials released by the developers state that ``there is currently no way to perform joint calling for somatic variant discovery`` \cite{GATKTeam2021a}. So while all methods in the GATK are considered a beta feature, the multi-sample mode needs to be used with care.

There are only two methods currently which have documented and published capabilities to jointly analyse tumour samples from the same patient to call somatic variants. The first \remove{one} is a specialised method built on a joint bayesian model for single nucleotide variants (SNV)s that occur in multiple samples called multiSNV \cite{Josephidou2015}. However, it has multiple shortcomings, \change{which make it not usable}{making it unusable} for our data. First, as the name suggests, the method can only jointly evaluate SNVs and completely ignores INDELs and structural variants, which would be acceptable for \change{the superior performance it provides}{its superior performance}. However, multiSNV was optimised only for WES \change{and not}{rather than} for the very deep WGS that is now widely available and forms a major part of this thesis. This mismatch of input types means exceptionally high runtimes on WGS data. Even with custom parallelisation that was attempted in this work, the predicted runtime for just one multi-sample patient would have been longer than \change{3}{three} years. This \add{shortcoming} shows that while multiSNV was a great step forward at the time, there is a real need for new methods to stem the tide of sequencing data available due to the ever-decreasing sequencing cost.

multiSNV had been the only software available for multi-sample analysis for almost five years, but during this work, superFreq \cite{Flensburg2020} was also published. It combines all standard analysis steps for tumour analysis, like quality assessment, variant calling, copy number analysis and clonal deconvolution, into one program and \change{is even able to}{can even} jointly analyse samples. However, similar to multiSNV, its focus during optimisation and development was on WES and RNAseq data, so when applied to WGS data, we could not find a server node with enough memory to execute the workflow.

This lack of methods then prompted us to develop a novel workflow to enable the analysis of high depth WGS, which we estimate to become more and more routine with the ever-dropping prices of sequencing. The following sections will show the development and validation of the joint variant calling methods as described in \textcite{Hollizeck2021} (\autoref{variantcalling-sec:publication}) and additional analysis on the impact of the joint variant calling on downstream multi-regional tumour analysis (\autoref{variantcalling-sec:downstream}), longitudinal sample analysis (\autoref{variantcalling-sec:longitudinal}) and clonal deconvolution (\autoref{variantcalling-sec:clonal}). The final section provides current information on the usage of the methods by others in the research community (\autoref{variantcalling-sec:usage}).

