\section{Introduction}
\label{variantcalling-sec:intro}

When I started exploring the somatic variant calling methods in the beginning of my PhD in 2018, I was surprised about the stark difference between germline and somatic variant calling methods. Where all "modern" germline variant callers have the built-in capability to joint call multiple samples, for example from family trios, virtually no somatic variant caller had this function. 

The joint analysis of smaller cohorts improves the performance of germline variant calling methods significantly, by allowing to assess technical artifacts, which might be unique for the individual sequencing machine or the researcher handling the DNA \cite{Schirmer2016,Stoler2021}. As certain parts of the genome are more problematic to sequence (\autoref{intro-sec:sequencing}) and map (\autoref{intro-sec:mapping}), a ``control`` sample can help to distinguish if a certain observed change occurs commonly, is a technical issue or in fact a real change.

For somatic variant calling, this concept has been adjusted in the genome analysis toolkit (GATK) \cite{BrianOConnor2020} to allow the use of panel of normals (PON), which contains frequently seen changes in healthy (``normal``) individuals analysed with the same sequencing technology \cite{GATKTeam2021}, but this is a post processing step of the analysis rather than a more intricate model like it is for the germline equivalent. Mutect2, which is the most recent somatic variant calling algorithm provided by the Broad institute, however also provides a multi-sample mode, for which all tumour samples need to be from the same patient, either longitudinal or spatial different \cite{GATKTeam2020}. This mode is hidden quite well and all tutorials published by the developers state that ``there is currently no way to perform joint calling for somatic variant discovery`` \cite{GATKTeam2021a}, so while all methods in the GATK are considered a beta feature, this seems, that development is not a priority.

There are only two methods currently, which have documented and published capabilities to jointly analyse tumour samples from the same patient to call somatic variants. The first one is a specialised method built on a joint bayesian model for SNVs to occur in multiple samples called multiSNV \cite{Josephidou2015}. However it has multiple shortcomings, which make it not useable for our data. First, as the name suggests, the method can only jointly evaluate SNVs and completely ignores indels and structural variants, which would be acceptable for the superior performance shown. However, multiSNV was optimised only for WES and not for the very deep WGS that is now available. This means exceptionally high runtimes. Even with custom parallelisation that was attempted in this work, the predicted runtime for just one multi sample patient would have been longer than 3 years. This again shows, that while multiSNV was a great step forward at the time, there is a real need for new methods to stem the tide of sequencing data available at low cost.

multiSNV has been the only software available for multi sample analysis, but only recently, during this work, superFreq \cite{Flensburg2020} was published. It combines all standard analysis steps for tumour analysis, like variant calling or clonal deconvolution, into one program and is even able to jointly analyse samples. However similar to multiSNV, its focus when optimising and developing was on WES and RNAseq data, so when applied to our data, we could not find a server with enough memory to execute the workflow.

This then prompted us to investigate possible workflows to enable the analysis of high depth WGS, which we estimate to become more and more normal, with the ever dropping prices of sequencing. The following sections will first show the publication and then discusses additional analysis done after the the publication of the manuscript (\autoref{variantcalling-sec:longitudinal}) and the impact of the joint analysis on downstream methods (\autoref{variantcalling-sec:downstream}).

