\acknowledgements{
\addtocontents{toc}{\vspace{1em}}  % Add a gap in the Contents, for aesthetics

The acknowledgements and the people to thank go here, don't forget to include your project advisor\ldots


Lots of figures in the introductory \autoref{ch:intro} were reated with the help of \href{https://BioRender.com}{\nolinkurl{BioRender.com}} 

\todo[inline]{think of where to put the package citations; Probably at the end as appendix}

\section*{Software and packages} This section is dedicated to all the software that usually gets uncited because they are "standard" or backbone 

Most analysis in a prototype state was done on a linux cluster running Centos 7 \cite{CentosProject2014} with Bash \cite{FSF2007} and due to the high amount of data, parallel \cite{Tange2011} was used of the multi-cpu architecture of HPCs.

\subsection*{R}
In depth data analysis and visualisation was done with R \cite{RCT2021} with the help of packages listed below.


Most of the parallelisation in R was performed with BiocParallel \cite{Morgan2020}, which is available through BiocManager \cite{Morgan2019}.

Colour shemes and manipulation was performed with colorspace \cite{Zeileis2009,Zeileis2020}.

Copynumber analysis was performed with sequenza \cite{Favero2015}, FACETS \cite{Shen2016,Seshan2018} and PURPLE \cite{Cameron2019a}. Some analysis was also directly performed with copynumber \cite{Nilsen2012,Nilsen2021}.

Variant effect prediction was performed with VEP \cite{McLaren2016}.

Table manipulation was performed with data.table \cite{Dowle2021}.

Violin plots were generated with vioplot \cite{Adler2020}.

Heatmaps and UpSet plots were generated with ComplexHeatmap \cite{Gu2016}

Phylogenetic analysis was performed with both ape \cite{Paradis2018} and phangorn \cite{Schliep2017} followed by dendextend \cite{Galili2015}.

Google sheets and its built in scripts were used to collect stats on docker pull requests and the data was then read in R through googlesheets4 \cite{Bryan2021}.

Additional libraries, which were used for a multitude of things are listed in no particular order below: Rsamtools \cite{Morgan2021}, GenomicRanges \cite{Lawrence2013}, optparse \cite{Davis2020}, VariantAnnotation \cite{Obenchain2014}, MultiAssayExperiment \cite{Ramos2017}, circlize \cite{Gu2014}, BioQC \cite{Zhang2017}, Biostrings \cite{Pages2020}, deconstructSigs \cite{Rosenthal2016}, BSgenome \cite{Pages2020a}, QDNAseq \cite{Scheinin2014}, RColorBrewer \cite{Neuwirth2014}, pheatmap \cite{Kolde2019}, ensemblVEP \cite{Obenchain2020}, stringdist \cite{vanderLoo2014}, Rsubread \cite{Liao2019}, svglite \cite{Wickham2021}, grImport \cite{Murrell2009}, XML \cite{TempleLang2020}, kableExtra \cite{Zhu2021}, lsa \cite{Wild2020}, irlba \cite{Baglama2019}, ggplot2 \cite{Wickham2016}

\todo[inline, color=green]{maybe itemize over just a blob}

\subsection*{python}
Analysis for \autoref{ch:mmf} was mostly done through python \cite{VanRossum2010} with the help of many different packages, which are listed here in no particular order: numpy \cite{Harris2020}, ncls \cite{Stovner2019}, pysam \cite{Heger2021,Bonfield2021,Danecek2021}, zarr \cite{Miles2021}, pandas \cite{McKinney2010,Reback2021}, quadprog \cite{McGibbon2021} as well as scipy \cite{Virtanen2020}.

\subsection*{latex}
Of course, finally the typesetting of the thesis itself was done with \LaTeX. With these additional packages in no particular order: babel, csquotes, lmodern, CrimsonPro, fontenc, xcolor, hhline, siunitx, biblatex, hyperref, quotchap, todonotes, float, afterpage, multicol, enumitem, array, tocloft, caption, appendix, xurl, graphicx, epstopdf, subfigure, booktabs, rotating and listings. 
The bese class is 'book' and all packages are available on CTAN and the source code is available at my GitHub repository \href{https://github.com/SebastianHollizeck/PhDThesis}{\nolinkurl{https://github.com/SebastianHollizeck/PhDThesis}}.

}

