\acknowledgements{
\addtocontents{toc}{\vspace{1em}}  % Add a gap in the Contents, for aesthetics

I first want to acknowledge the Wurundjeri people of the Kulin nation, the traditional custodians of the land on which my work was conducted. I want to pay respects to their elders: past, present and emerging and all other elders that might happen to read this work. In my time in Australia I was lucky to get a glimpse of this mysterious and special continent and country through their eyes and I am grateful for their ongoing work in keeping the legends and teachings alive.

\section*{People} 
\todo[inline,color=red]{fill this in with people}

% First of all I want to thank Imran House, who got me excited for academical science again. When I finished my Master's degree I was convinced that I would neither need nor want to get a PhD, but after working with him and experiencing his excitement about cutting edge research and his attitude towards academia I opened up to the idea of a PhD. And then he even helped me get a position and shoed me around this beautiful country and it's many loveable customs.

%When I said Imran helped me get a PhD position I mean he put me in contact with the lovely Sarah Ftouni, who then helped me every step of the way as if I was her own brother, even though I know how much of a pain I can be. She helped me through all this time with even the dumbest questions and requests with barely any complaints. A geniune treasure and great lab manager.

%While I constantly get told that my english is really good (for a german) I did not actually qualify immediatly for the english requirements set by the \University, but alone came Caroline Owen and just paved the way. Without her I would probably still sit in Germany and trying to figure out the intricacies of both the VISA and stipdend applications. When she passed the hat over to Erika Cretney, Erika and her team did everything in their power to help me through the issues of an ever shrinking staff at the university and COVID-19 regulations. So while Caroline made me coming to Australia possible, I want to thank the whole research education team for their continous efforts.

%I also want to extend my thanks here to my previous group leader and employer Christoph Klein, who I had as a professional reference on my application for both my PhD position as well as my stipend. When he was only asked as few generic questions in a form, he insisted on writing a free form letter of recommendation, as he felt he could not point out my qualities in the limited form.

%While Sarah-Jane Dawson has helped and guided and mentored me in many ways, for which I am very grateful, the first ever expression of trust in me was her offering to pay my stipend if I was not successful in my application with the university. It was such a massive sign of faith in me and my abilities I could not believe it in the beginning. But over the last years, Sarah-Jane has always shown me that my interests and wishes are important to her. She supported and challenged me to grow both as a person as well as a scientist. Her ability to conceptualise and prioritise were a great inspiration.

%With Dineika Chandrananda I struck a real jackpot as a supervisor. Even though she only became my official supervisor after a while, she helped me orient myself and offered invaluable advice and patience when discussing methodological ideas. She always had an open ear for my issues and often urged me to take a break and come back refreshed. I am very thankful to have had a supervisor that cared at least as much about my work as my mental health.

%Even though my last supervisor Ben Solomon is a very busy person, he always had time to meet and discuss patient data or research questions. He was always advcating for me and my professional development, even at a possible detriment to his own. I even had the chance to meet Ben's lovely family, which again showed that it was not just about my work. He is a great scientist and doctor working hard to make the world a better place.

%I also want to thank Mark Dawson, who even though he was not officially part of the work in this thesis, he did support and offer advice. Exactly like Sarah-Jane he believed in me and my skills. He was always interested in my progress and valued my opinion.

%



\section*{Software and packages} 

This section is dedicated to all the software that usually gets un-cited because they are ``standard`` or backbone.

Lots of figures in the introductory \autoref{ch:intro} were created with the help of \href{https://BioRender.com}{\nolinkurl{BioRender.com}} 


Most analysis in a prototype state was done on a linux cluster running Centos 7 \cite{CentosProject2014} with Bash \cite{FSF2007} and due to the high amount of data, parallel \cite{Tange2011} was used of the multi-cpu architecture of HPCs.

\subsection*{R}
In depth data analysis and visualisation was done with R \cite{RCT2021} with the help of packages listed below.


Most of the parallelisation in R was performed with BiocParallel \cite{Morgan2020}, which is available through BiocManager \cite{Morgan2019}.

Colour scheme selection and manipulation was performed with colorspace \cite{Zeileis2009,Zeileis2020}.

Copy number analysis was performed with sequenza \cite{Favero2015}, FACETS \cite{Shen2016,Seshan2018} and PURPLE \cite{Cameron2019a}. Some analysis was also directly performed with copynumber \cite{Nilsen2012,Nilsen2021}.

Variant effect prediction was performed with VEP \cite{McLaren2016}.

Table manipulation was performed with data.table \cite{Dowle2021}.

Violin plots were generated with vioplot \cite{Adler2020}.

Heatmaps and UpSet plots were generated with ComplexHeatmap \cite{Gu2016}

Phylogenetic analysis was performed with both ape \cite{Paradis2018} and phangorn \cite{Schliep2017} followed by dendextend \cite{Galili2015}.

Google sheets and its built in scripts were used to collect stats on docker pull requests and the data was then read in R through googlesheets4 \cite{Bryan2021}.

Additional libraries, which were used for a multitude of things are listed in no particular order below: Rsamtools \cite{Morgan2021}, GenomicRanges \cite{Lawrence2013}, optparse \cite{Davis2020}, VariantAnnotation \cite{Obenchain2014}, MultiAssayExperiment \cite{Ramos2017}, circlize \cite{Gu2014}, BioQC \cite{Zhang2017}, Biostrings \cite{Pages2020}, deconstructSigs \cite{Rosenthal2016}, BSgenome \cite{Pages2020a}, QDNAseq \cite{Scheinin2014}, RColorBrewer \cite{Neuwirth2014}, pheatmap \cite{Kolde2019}, ensemblVEP \cite{Obenchain2020}, stringdist \cite{vanderLoo2014}, Rsubread \cite{Liao2019}, svglite \cite{Wickham2021}, grImport \cite{Murrell2009}, XML \cite{TempleLang2020}, kableExtra \cite{Zhu2021}, lsa \cite{Wild2020}, irlba \cite{Baglama2019}, ggplot2 \cite{Wickham2016}


\subsection*{python}
Analysis for \autoref{ch:mmf} was mostly done through python \cite{VanRossum2010} with the help of many different packages, which are listed here in no particular order: numpy \cite{Harris2020}, ncls \cite{Stovner2019}, pysam \cite{Heger2021,Bonfield2021,Danecek2021}, zarr \cite{Miles2021}, pandas \cite{McKinney2010,Reback2021}, quadprog \cite{McGibbon2021} as well as scipy \cite{Virtanen2020}.

\subsection*{latex}
Of course, finally the typesetting of the thesis itself was done with \LaTeX. With these additional packages in no particular order: babel, csquotes, lmodern, CrimsonPro, fontenc, xcolor, hhline, siunitx, biblatex, hyperref, quotchap, todonotes, float, afterpage, multicol, enumitem, array, tocloft, caption, appendix, xurl, graphicx, epstopdf, subfigure, booktabs, rotating and listings. 
The base class is 'book' and all packages are available on CTAN and the source code is available at my GitHub repository \href{https://github.com/SebastianHollizeck/PhDThesis}{\nolinkurl{https://github.com/SebastianHollizeck/PhDThesis}}.

}

