%
% Guidelines as of 2019/06/04
% https://gradresearch.unimelb.edu.au/__data/assets/pdf_file/0004/2027839/Preparation-of-GR-theses-rules.pdf
%
\Preface{

\addtocontents{toc}{\vspace{1em}}  % Add a gap in the Contents, for aesthetics

This preface includes a summary of all chapters in this work as well as a comprehensive summary of my contributions and everyone else's contribution. This is a thesis \textit{with} publications and each publication included in a chapter is shown here.



\textbf{Hollizeck S.}, Wong S.Q., Solomon B., Chandrananda D.\footnote{These authors contributed equally and are considered shared last.}, Dawson S-J.\footnotemark[\value{footnote}] \textbf{``Custom workflows to improve joint variant calling from multiple related tumour samples: FreeBayesSomatic and Strelka2Pass``} \textit{Bioinformatics.} 2021. DOI: \href{https://doi.org/10.1093/bioinformatics/btab606}{\nolinkurl{10.1093/bioinformatics/btab606}}


%%%%%%%%%%%%%%%%%%%%%%%

\paragraph*{\hyperref[ch:intro]{Chapter~\ref*{ch:intro}}:} The Introduction is an original work providing background and overview relevant to understanding the thesis and its relevance to the field. It includes an introduction to DNA, ctDNA, DNA sequencing, somatic variant calling and tumour heterogeneity.


%%%%%%%%%%%%%%%%%%%%%%%

\paragraph*{\hyperref[ch:variantcalling]{Chapter~\ref*{ch:variantcalling}}:} The chapter ``Joint somatic variant calling`` is an original work describing two workflows for the joint analysis of multiple related tumour samples and has been published in \textit{Bioinformatics} as "Custom workflows to improve joint variant calling from multiple related tumour samples: FreeBayesSomatic and Strelka2Pass" on 21$^{st}$ September 2021. In addition to the published analysis, I have added longitudinal analysis and its evaluation as well as the impact of this new method on other downstream analysis, like phylogenetic reconstruction and clonal deconvolution.

Contributions for this chapter:
\begin{itemize}
	\item I conceptualised the work
	\item I implemented the workflows and containerised all required tools
	\item I performed the data simulation
	\item I performed the analysis presented in the publication
	
%\begingroup\interlinepenalty=10000
	\item I wrote the draft of the manuscript and performed revisions
	\item Dineika Chandrananda (D.C.) and Sarah-Jane Dawson (S-J.D). provided advice in planning and writing the manuscript
	\item D.C. provided guidance for method development
	\item S-J.D. provided guidance for method evaluation
	\item Stephen Wong (S.W.) performed the targeted amplicon validation
	\item S.W. and  Ben Solomon (B.S.) read the draft manuscript and provided feedback
	\item B.S. provided clinical expertise for human data
	
%\endgroup
\end{itemize}


%%%%%%%%%%%%%%%%%%%%%%%

\paragraph*{\hyperref[ch:cascade]{Chapter~\ref*{ch:cascade}}:} 
In this chapter the analysis of five lung cancer patients is described with regards to intra- and inter-patient tumour heterogeneity. A special focus was put on sample and clonal relationships using the variant calling methods from Chapter~\ref*{ch:variantcalling}. We also developed a phylogenetic reconstruction method based on mitochondrial variants, which allowed a different avenue for insight into metastatic seeding and timing. Parts of this analysis has been published\cite{Burr2019,Solomon2020}, but the publications are not included in this thesis, as I did not contribute more than 50\% of the work of the articles. 


Contributions for this chapter:
\begin{itemize}
	\item I analysed all data
	\item I generated all visualisations
	\item I wrote the draft
	\item I implemented the mitochondrial phylogeny reconstruction method
	\item S-J.D., D.C., and Mark Dawson (M.D.) conceptualised the mitochondrial phylogeny reconstruction
	
%\begingroup\interlinepenalty=10000

	\item S-J.D., D.C. and  B.S. read the draft manuscript and provided feedback
	\item Lavinia Tan (L.T.) and B.S. provided clinical expertise for human data
	
%\endgroup
\end{itemize}

\paragraph*{\hyperref[ch:mmf]{Chapter~\ref*{ch:mmf}}:}
The MisMatchFinder analysis method described in this chapter is an original work using a read-centric variant calling approach to detect tumour somatic mutational signatures from low coverage sequencing data. The work describes individual design decisions and shows the performance of the method in multiple datasets. This work is unpublished and has not yet been submitted for publication.

Contributions for this chapter:
\begin{itemize}
   	\item I conceptualised the work with input from S.W.
	\item I analysed all data
	\item I generated all visualisations
	\item I implemented the method.
	\item I wrote the draft
	\item D.C. provided guidance and support for method development
	\item S-J.D. and D.C. read the draft and provided feedback
	\item Lavinia Tan (L.T.) and B.S. provided clinical expertise for human data
	
%\endgroup
\end{itemize}

\paragraph*{\hyperref[ch:conclusion]{Chapter~\ref*{ch:conclusion}}:}
The conclusion chapter places the thesis in the wider field of existing methods to assess tumour heterogeneity and outlines future directions of the field.

\cleardoublepage

\paragraph*{Other publications} These publications I have contributed to in my candidature, but they are not presented in this work

Burr M.L., Sparbier C.E., Chan K.L., Chan Y-C., Kersbergen A., Lam E.Y.N., Azidis-Yates E., Vassiliadis D., Bell C.C., Gilan O., Jackson S., Tan L., Wong S.Q., \textbf{Hollizeck S.}, Michalak E.M., Siddle H.V.,  McCabe M.T., Prinjha R.K., Guerra G.R., Solomon B.J., Sandhu S.,  Dawson S-J., Beavis P.A., Tothill R.W., Cullinane C., Lehner P.J., Sutherland K.D., Dawson M.A. \textbf{``An evolutionarily conserved function of polycomb silences the MHC class I antigen presentation pathway and enables immune evasion in cancer``} \textit{Cancer cell.} 2019. DOI: \href{https://doi.org/10.1016/j.ccell.2019.08.008}{10.1016/j.ccell.2019.08.008}

Solomon B.J.\footnote{These authors contributed equally and are considered shared first.}., Tan L.\footnotemark[\value{footnote}], Lin J.J.\footnotemark[\value{footnote}], Wong S.Q.\footnotemark[\value{footnote}], \textbf{Hollizeck S.}\footnotemark[\value{footnote}], Ebata K., Tuch B.B., Yoda S., Gainor J.F., Lecia V. Sequist L.V., Oxnard G.R., Gautschi O., Drilon A., Subbiah V., Khoo C., Zhu E.Y., Nguyen M., Henry D., Condroski K.R., Kolakowski G.R., Gomez E., Ballard J., Metcalf A.T., Blake J.F., Dawson S-J., Blosser W., Stancato L.F., Brandhuber B.J., Andrews S., Robinson B.G., Rothenberg S.M
\textbf{``RET Solvent Front Mutations Mediate Acquired Resistance to Selective RET Inhibition in RET-Driven Malignancies``}
\textit{Journal of Thoracic Oncology.} 2020. DOI: \href{https://doi.org/10.1016/j.jtho.2020.01.006}{\nolinkurl{10.1016/j.jtho.2020.01.006}}

Fennell K.A.\footnotemark[\value{footnote}], Vassiliadis D.\footnotemark[\value{footnote}], Lam E.Y., Martelotto L.G., Balic J.J., \textbf{Hollizeck S.}, Weber T.S., Semple T., Wang Q., Miles D.C., MacPherson L., Chan Y-C. Guirguis A.A., Kats L.M., Wong E.S., Dawson S-J., Naik S.H., Dawson M.A. \textbf{``Non-genetic determinants of malignant clonal fitness at single cell resolution``} \textit{Nature.} 2021 DOI: \href{https://doi.org/10.1038/s41586-021-04206-7}{\nolinkurl{10.1038/s41586-021-04206-7}}





}
